### PR TITLE
[16.0][MIG] stock_split_picking

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard Migration.

### CODE REVIEW

**Review by dannyadair — APPROVED** [review]
Looks good to me

As per https://github.com/OCA/OpenUpgrade/blob/16.0/openupgrade_scripts/scripts/stock/16.0.1.1/upgrade_analysis.txt#L57

stock        / stock.move.line          / product_qty (float)           : DEL
stock        / stock.move.line          / product_uom_qty (float)       : DEL required
stock        / stock.move.line          / reserved_qty (float)          : NEW isfunction: function, stored
stock        / stock.move.line          / reserved_uom_qty (float)      : NEW required, hasdefault: default

and

stock        / stock.picking            / move_ids (one2many)           : NEW relation: stock.move
stock        / stock.picking            / move_lines (one2many)         : DEL relation: stock.move

File: stock_split_picking/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        )
+        self.message_post(
+            body=_(
+                'The backorder <a href="#" '
+                'data-oe-model="stock.picking" '
+                'data-oe-id="%(id)d">%(name)s</a> has been created.'
+            ),
+            id=backorder_picking.id,
+            name=backorder_picking.name,
+        )
```
Comment (AungKoKoLin1997) [inline stock_split_picking/models/stock_picking.py]:
```suggestion
        self.message_post(
            body=_(
                'The backorder %s has been created.'
            , backorder_picking._get_html_link())
        )
```

**Review by AungKoKoLin1997 — CHANGES_REQUESTED** [review]

**Review by ioans73 — APPROVED** [review]
Functional review

File: stock_split_picking/models/stock_picking.py (hunk 1 above)
Comment (rousseldenis) [reply stock_split_picking/models/stock_picking.py]: @ps-tubtim

Comment (ps-tubtim) [reply stock_split_picking/models/stock_picking.py]: @AungKoKoLin1997 @rousseldenis  I already change it.

**Review by AungKoKoLin1997 — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**